DjangoTut/settings.py

```diff
@@ -116,15 +128,24 @@
 
 USE_TZ = True
 
+LOCALE_PATHS = (
+    os.path.join(BASE_DIR, 'locale'),
+)
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
 STATIC_URL = '/static/'
-STATICFILES_DIRS = [
-    os.path.join(BASE_DIR, "static"),
-    '/var/www/static/',
-]
 
 LOGIN_REDIRECT_URL = 'home'
 LOGOUT_REDIRECT_URL = 'home'
+
+AUTHENTICATION_BACKENDS = (
+    "django.contrib.auth.backends.ModelBackend",
+    "polls.auth.AuthenticationEmailBackend",
+)
+
+AUTH_USER_MODEL = 'polls.UserProfile'
```

Comment:
ogólnie ja bym stworzył moduł 'core' albo 'auth' i tam trzymał takie ogólne rzeczy. tam można stworzyć ten model User, dodać standordowo logowania itd. a już jak będziesz robił połączenie z tym api to zrobisz dodatkowy moduł 'youtube' naprzykład